liferay-echo/liferay-portal
# LPS-193747 Replace AUI with js/react in server-admin-web module

Motivation
=======
The motivation behind this ticket is to migrate from AUI old javascript files to functional/react js components.
https://liferay.atlassian.net/browse/LPS-193747

Proposed Solution
========
What I did to solve this problem is to change the file structure and to use equivalents methods in new js structure

Steps to Verify
========
1. Go to Server Administration and check that all the action buttons work correctly
